opensearch/src/main/java/org/opensearch/sql/opensearch/data/type/OpenSearchGeoPointType.java

```diff
@@ -22,8 +26,15 @@ public class OpenSearchGeoPointType extends OpenSearchDataType {
   private OpenSearchGeoPointType() {
     super(MappingType.GeoPoint);
     exprCoreType = UNKNOWN;
+    this.properties = new HashMap<>();
+    this.properties.put("lat", new OpenSearchDataType(ExprCoreType.DOUBLE));
+    this.properties.put("lon", new OpenSearchDataType(ExprCoreType.DOUBLE));
   }
 
+  @Getter
+  @EqualsAndHashCode.Exclude
+  Map<String, OpenSearchDataType> properties;
```

Comment:
Probably you can make `properties` in the base class `protected` instead

Reply:
Removed to just use base class properties.